moveit_setup_assistant/moveit_setup_framework/include/moveit_setup_framework/templates.hpp

```diff
@@ -69,7 +69,7 @@ class TemplatedGeneratedFile : public GeneratedFile
 
   bool write() override;
 
-  static std::vector<TemplateVariable> variables_;
+  static std::vector<TemplateVariable> variables;
```

Comment:
are we sure that we want to drop `_` for static member variables? I've just found the same issue here https://github.com/ros-planning/moveit2/actions/runs/4862228141/jobs/8670864807?pr=2119#step:12:9677, but I'm not convinced that it's actually a good format rule

Reply:
I don't have a strong opinion about this so I can definitely revert it, but it is interesting that this CI error seems to pop up randomly. This PR and the other did not touch the files where it occurs